src/Data/CurrentUser/CurrentUserDataItem.ts

```diff
@@ -4,15 +4,22 @@ import { ensureString } from '../../utils/ensureString'
 import { isOptionalString } from '../../utils/isOptionalString'
 import { neoletterClient } from '../neoletterClient'
 import { pisaClient } from '../pisaClient'
+import { errorToast } from './errorToast'
 
 export const CurrentUser = provideDataItem('CurrentUser', {
   async get() {
     const user = await load(currentUser)
     if (!user) return {}
 
-    const neoletterProfile = await neoletterClient().get('my/profile')
-    if (!isNeoletterData(neoletterProfile)) {
-      throw new Error('Neoletter data is not in the expected format')
+    let neoletterProfile
+    try {
+      neoletterProfile = await neoletterClient().get('my/profile')
+      if (!isNeoletterData(neoletterProfile)) {
+        throw new Error('Neoletter data is not in the expected format')
+      }
+    } catch (error) {
+      errorToast('Unable to connect to Neoletter', error)
```

Comment:
* [ ] The important message is that "A technical error occurred". Not sure if it should be passed as an argument. Currently all error toasts are technical errors, so we could move this to the helper instead of repeating it.
* [ ] Given the main message above I'm not sure if we have to pass in both an additional message and the error. It depends on how likely these errors are and how likely a custom message will help the user. "Unable to connect" may even be misleading. Maybe just `error.message` (which is already customizable if needed, like inside the try block) is good enough for technical breakdowns?